src/Controls/index.js

```diff
@@ -25,22 +26,28 @@ const AddWMSLayers = (overlays, WMSLayerGroup, mapRef) => {
   return overlays
 }
 
-const AddLayerControlsOverlays = (Config, DynamicLayerGroup, WMSLayerGroup, mapRef) => {
+const AddLayerControlsOverlays = (DynamicData, DynamicLayerGroup, WMSLayerGroup, mapRef) => {
   let overlays = {}
-  if (Config.DynamicData !== undefined) {
-    if (Config.DynamicData.some(layer => layer.displayInOverlay)) {
-      Config.DynamicData.map(layer => {
-        if (layer.displayInOverlay) {
-          overlays[layer.key] = DynamicLayerGroup[layer.key]
-        }
-        if (layer.visibleByDefault) {
-          DynamicLayerGroup[layer.key].addTo(mapRef)
+  if (DynamicData == null) {
+    return AddWMSLayers(overlays, WMSLayerGroup, mapRef)
+  }
+
+  for (var x = 0; x < DynamicData.length; x++) {
+    let layer = DynamicData[x]
+    if (layer.displayInOverlay) {
+      if (!layer.group) {
+        overlays[layer.key] = DynamicLayerGroup[layer.key]
+      } else {
+        let groupKey = `<span style=font-weight:bold;>${layer.group}</span>`
```

Comment:
Do we need to add in the HTML and CSS here for the Array key? can this not be added in later and can we use the GDS CSS classes to style the span not inline styling?